### DISCUSSION TITLE
How to prevent a notifications every time I create an issue

### DISCUSSION THREAD

**Discussion by robopzet** [community/community#108404] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Just creating an issue adds a notification on the site. Even after I unsubscribed from every repository.

How can I turn off these notifications, or better, define the situations I do want to get notified. Like only when someone used my @handle, when I opted-in to be notified for every new comment on an issue or when the status changed,

**Comment by jlord** [comment]
👋 Even if you are unsubscribed to repository, if you author an issue you are subscribed to that issue by default. But you can manually click the `unsubscribe` button on that issue if you don't want to be notified after you create it. Hope that helps!
